lib/modules/platform/gerrit/scm.ts

```diff
@@ -109,7 +109,7 @@ export class GerritScm extends DefaultGitScm {
       typeof commit.message === 'string' ? [commit.message] : commit.message;
     commit.message = [
       ...origMsg,
-      `Change-Id: ${existingChange?.change_id ?? generateChangeId()}`,
+      `Renovate-Source-Branch: ${commit.branchName}\nChange-Id: ${existingChange?.change_id ?? generateChangeId()}`,
```

Comment:
Here is one of my own notes:

If we make this shorter, there is a lesser chance that this line will offend line length rules of commit message linters.

For example: `Source-Branch` or `Renovate-Branch`.

Reply:
I think it's important to decide/finalize this now?

Reply:
Absolutely.

Reply:
@NiasSt90 what's your opinion about this?

Reply:
For now I renamed to `Renovate-Branch`.